admin/client/src/state/formData/README.md

```diff
@@ -0,0 +1,9 @@
+# Form Data state
+
+Uses `formData` store.
+
+Manages state associated with the FormBuilderLoader.
+
+This sets form data that does not quite belong in schema, as it is client-side generated data, as opposed to server-side generated for schema.
```

Comment:
Just to close this off, Chris has now implement this as `metadata` and `stateOverride` keys.